app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    count = 0
+    v_error = 0
+    m_error = 0
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError:
+            v_error += 1
+            count += 1
```

Comment:
u can make return "All friends should be vaccinated" here - no need of this counters